src/store/modules/shipment/actions.ts

```diff
@@ -18,9 +18,8 @@ const actions: ActionTree<ShipmentState, RootState> = {
         "statusId": "PURCH_SHIP_SHIPPED"
       })
       if (resp.status === 200 && resp.data.count> 0 && !hasError(resp)) {
-        let shipments = resp.data.docs;
-
-        if (payload.viewIndex && payload.viewIndex > 0) shipments = state.shipments.list.concat(shipments)
+        const shipments = resp.data.docs;
+        if (!shipments.length) showToast(translate("No more shipments found")); 
```

Comment:
The if condition is not needed, as we are already checking for count variable.

Reply:
Sir, I have added this if condition because if we will continusally click on load more shipment button app will show a toast when shipments are over.

Reply:
I think instead of showing a toast when there are no more shipments, we should hide the `Load more shipments` button. WDYT? @dt2patel

Reply:
no I think we should keep the button incase the user wants to retry loading the shipments